app/views/companies/_form.html.haml

```diff
@@ -29,6 +29,12 @@
                 = f.text_area :description, rows: 4, class: "form-control", placeholder: "Company description"
             .form-group
               .control-label.col-md-2
+                = f.label :logo
```

Comment:
@LevinGeorge For the label always keep it the same name as the column in the database. If you want to pass in a custom word to appear on the logo you can do it by passing second paramater to label, so for e.g ' f.label :avatar, "Logo" '.